src/plugins/projectexplorer/projectvcsstatus.cpp

```diff
@@ -1,24 +1,27 @@
 // License: GPL
-#include "coreplugin/iversioncontrol.h"
-#include "coreplugin/vcsmanager.h"
-#include "coreplugin/editormanager/editormanager.h"
-#include "projecttree.h"
 #include "projectvcsstatus.h"
-#include "session.h"
 #include "project.h"
+#include "projecttree.h"
+#include "session.h"
+
+#include <coreplugin/editormanager/editormanager.h>
+#include <coreplugin/vcsmanager.h>
```

Comment:
it seems you slightly changed the includes here… I hope you can avoid this, to reduce risk of future merge conflicts.